.oxlintrc.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "./node_modules/oxlint/configuration_schema.json",
+  "plugins": ["vue", "unicorn"],
```

Comment:
You might want to enable the eslint plugin too here for core eslint rules (or further down)

Reply:
Ok thanks I'll try! This plugins were injected by `oxlint-migrate`

Reply:
cc @Sysix (just FYI)